bai-tap-buoi-9/main.js

```diff
@@ -1 +1,151 @@
-const 
\ No newline at end of file
+// Hàm kiểm tra số nguyên
+function checkNumber(number) {
+    if (parseInt(number)==number) {
+        return true;
+    }
+    else {
+        return false;
+    }
+}
+// Bài 1:
+const realSalary = (money) => {
+    if (money >= 15000000) {
+        console.log('Thuế thu nhập: ' + money*0.3)
+        console.log('Lương ròng: '+money*0.7)
+    }
+    else if (money >= 7000000 && money < 15000000) {
```

Comment:
```suggestion
    else if (money >= 7000000) {
```
Bên trên đã kiểm tra salary >= 15m rồi, xuống dưới cần gì kiểm tra nữa, tương tự cho các phần bên dưới